sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTraceExporter.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    /// <summary>
+    /// TODO.
```

Comment:
Please fill in these summary's before merging. We've had issues with devs forgetting.